firefox-ios/firefox-ios-tests/Tests/XCUITests/PocketTests.swift

```diff
@@ -5,8 +5,49 @@
 import XCTest
 
 class PocketTests: BaseTestCase {
+    enum SwipeDirection {
+        case up, down, left, right
+    }
+
+    func validatePocketStoriesCount() {
+        let numPocketStories = app.collectionViews.containing(
+            .cell,
+            identifier: AccessibilityIdentifiers.FirefoxHomepage.Pocket.itemCell
+        ).children(matching: .cell).count - 1
+
+        if iPad() {
+            XCTAssertTrue(numPocketStories > 7, "Expected at least 8 stories on iPad.")
+        } else {
+            XCTAssertTrue(numPocketStories > 6, "Expected at least 7 stories on iPhone.")
+        }
+    }
+
+    func togglePocket(shouldEnable: Bool) {
+        navigator.performAction(shouldEnable ? Action.TogglePocketInNewTab : Action.TogglePocketInNewTab)
+        navigator.goto(NewTabScreen)
+    }
+
+    func scrollToElement(_ element: XCUIElement, direction: SwipeDirection, maxSwipes: Int) {
```

Comment:
The syntax to set default values may be `maxSwipes: Int? = 5`.

Reply:
Updated the code with `maxSwipes: Int? = 5`. In this case the optional nature of maxSwipes (Int?) is no longer necessary because the default value is sufficient for most use cases and we can avoid any issues with optional unwrapping.